docs/features/automation-rules.md

```diff
@@ -187,6 +187,12 @@ fields: {
 }
 ```
 
+Creating a Jira ticket with a parent example:
```

Comment:
update to "In the following Jira example, the ticket created from Insights will be associated as a sub-issue to an existing parent issue:"

Reply:
done